transformer_engine/pytorch/csrc/ts_fp8_op.cpp

```diff
@@ -92,11 +92,11 @@ at::Tensor te_gemm_ts(at::Tensor A,
   bool accumulate_arg = static_cast<bool>(accumulate);
   bool use_split_accumulator_arg = static_cast<bool>(use_split_accumulator);
 
-  at::Tensor A_scale_inverse_arg = A_scale_inverse.clone();
+  at::Tensor A_scale_inverse_arg = A_scale_inverse;
   if (A_scale_inverse.numel())
     A_scale_inverse_arg = A_scale_inverse[A_fp8_tensor];
 
-  at::Tensor B_scale_inverse_arg = B_scale_inverse.clone();
+  at::Tensor B_scale_inverse_arg = B_scale_inverse;
```

Comment:
Can't we just use the original value instead? This just makes the code harder to read for no reason.

Reply:
Sure we could, we still need to check that tensor is not empty though.